source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -78,26 +76,26 @@ async function init(): Promise<void> {
 			const branchName = branchCompareLink.closest('[branch]')!.getAttribute('branch')!;
 			const prInfo = associatedPullRequests[branchName];
 			if (prInfo) {
-				const StateIcon = prInfo.state === 'MERGED' ? GitMergeIcon : GitPullRequestIcon;
+				const StateIcon = stateIcon[prInfo.state];
+				const state = upperCaseFirst(prInfo.state);
 
 				branchCompareLink.replaceWith(
 					<div className="d-inline-block text-right ml-3">
 						<a
 							data-issue-and-pr-hovercards-enabled
 							href={prInfo.url}
-							className="muted-link Link--muted"
 							data-hovercard-type="pull_request"
 							data-hovercard-url={prInfo.url + '/hovercard'}
 						>
-							#{prInfo.number}{' '}
+							#{prInfo.number}
 						</a>
-						<a
-							className={`State ${stateClass[prInfo.state]} State--small ml-1 no-underline`}
-							title={`Status: ${upperCaseFirst(prInfo.state)}`}
-							href={prInfo.url}
+						{' '}
+						<span
+							className={`State State--${prInfo.state.toLowerCase()} State--small ml-1`}
+							title={`Status: ${state}`}
```

Comment:
Can the classes be moved onto the stateIcon itself? Also I'm not sure we need a title here since there's already the text after it

Reply:
Github has both

Reply:
Yes and we do [remove tooltips that are useless](https://github.com/sindresorhus/refined-github/blob/main/source/features/hide-obvious-tooltips.css) like this one

<img width="138" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/132982412-4f58522e-8dee-441f-85fa-6bcf39483660.png">

Reply:
> Can the classes be moved onto the stateIcon itself?

This is how GitHub do it, not sure if saving a span can bring any benefit...

> Also I'm not sure we need a title here since there's already the text after it

I'm just following GitHub, though I agree it's redundant. Removed now.